lib/config/loader.js

```diff
@@ -196,6 +196,53 @@ module.exports = function (userCfgPath) {
 					default: userConfig.login && userConfig.login.twitch && userConfig.login.twitch.enabled ? null : []
 				}
 			},
+			discord: {
+				enabled: {
+					doc: 'Whether to enable Discord authentication.',
+					format: Boolean,
+					default: false
+				},
+				clientID: {
+					doc: 'A Discord application ClientID https://discord.com/developers/applications/',
+					format: String,
+
+					// This will throw if the user does not provide a value, but only if Discord auth is enabled.
+					default: userConfig.login && userConfig.login.discord && userConfig.login.discord.enabled ? null : ''
+				},
+				clientSecret: {
+					doc: 'A Discord application ClientSecret https://discord.com/developers/applications/',
+					format: String,
+
+					// This will throw if the user does not provide a value, but only if Discord auth is enabled.
+					default: userConfig.login && userConfig.login.discord && userConfig.login.discord.enabled ? null : ''
+				},
+				scope: {
+					doc: 'A space-separated string of Discord application permissions.',
+					format: String,
+					default: 'identify'
+				},
+				allowedIDs: {
```

Comment:
Should be allowedUserIDs instead to distinct from guilds properties below.

Reply:
Done!